crl/updater/updater.go

```diff
@@ -181,31 +181,30 @@ func (cu *crlUpdater) Run(ctx context.Context) error {
 // error at the end.
 func (cu *crlUpdater) Tick(ctx context.Context, atTime time.Time) (err error) {
 	defer func() {
-		// Each return statement in this function assigns its returned value to the
```

Comment:
It looks like you're moving away from the "deferred assignment to named return variable" trick. Want to un-name the return variable while you're at it?

Reply:
oh right, got it.